src/main/java/edu/kis/powp/jobs2d/command/manager/DriverCommandManager.java

```diff
@@ -28,31 +29,12 @@ public synchronized void setCurrentCommand(DriverCommand commandList) {
 
 	/**
 	 * Set current command.
-	 * 
+	 *
 	 * @param commandList list of commands representing a compound command.
 	 * @param name        name of the command.
 	 */
 	public synchronized void setCurrentCommand(List<DriverCommand> commandList, String name) {
-		setCurrentCommand(new ICompoundCommand() {
-
-			List<DriverCommand> driverCommands = commandList;
-
-			@Override
-			public void execute(Job2dDriver driver) {
-				driverCommands.forEach((c) -> c.execute(driver));
-			}
-
-			@Override
-			public Iterator<DriverCommand> iterator() {
-				return driverCommands.iterator();
-			}
-
-			@Override
-			public String toString() {
-				return name;
-			}
-		});
-
+		setCurrentCommand(new CompoundCommand(commandList));
```

Comment:
Watch out: there might be conflicts with other merges (it depends on who will merge first ;) )